File: migration/languages.py
from pymongo import MongoClient

client = MongoClient("URI")
db = client["multilingual-language"]
languages_collection = db["languages"]

languages_to_add = [
    {
        "name": "German",
    },
    {
        "name": "English",
    },
    {
        "name": "Spanish",
    },
    {
        "name": "Italian",
    }
]

try:
    result = languages_collection.insert_many(languages_to_add)
    print("Languages added with IDs:", result.inserted_ids)
except Exception as e:
    print("An error occurred:", e)

File: src/lib/languages.ts
import db from "./db";

export async function getLanguages() {
  return db.languages.findMany();
}

export async function getGermanLanguage() {
  return db.languages.findFirst({
    where: {
      name: "German",
    },
  });
}

export async function getLanguageByName(language: string) {
  return db.languages.findFirst({
    where: {
      name: language,
    },
  });
}
